data/change-streams/change-streams-disambiguatedPaths.json

```diff
@@ -26,6 +26,7 @@
   "runOnRequirements": [
     {
       "minServerVersion": "6.1.0",
+      "serverless": "forbid",
```

Comment:
This is to prevent failures in running change streams on serverless:

```
=== RUN   TestUnifiedSpec/change-streams/change-streams-disambiguatedPaths.json/disambiguatedPaths_is_not_present_when_showExpandedEvents_is_false/unset
    unified_spec_runner.go:130: error running operation "createChangeStream" at index 1: error verification failed: expected no error, but got (AtlasError) $changeStream is not allowed or the syntax is incorrect, see the Atlas documentation for more information
--- FAIL: TestUnifiedSpec (1.66s)
    --- FAIL: TestUnifiedSpec/change-streams (1.54s)
        --- FAIL: TestUnifiedSpec/change-streams/change-streams-disambiguatedPaths.json (1.54s)
            --- FAIL: TestUnifiedSpec/change-streams/change-streams-disambiguatedPaths.json/disambiguatedPaths_is_not_present_when_showExpandedEvents_is_false/unset (1.54s)
FAIL
```

Reply:
I expect other drivers may encounter the same error. I suggest opening a PR to the specifications repo.

In general, the copies of specification tests in the driver should match the specifications repo, so it is easier to resync them later.

Reply:
I will revert these changes and open a new GODRIVER PR for the Go Driver split of DRIVERS-2431 .